src/components/notification/__tests__/Notification.test.js

```diff
@@ -75,101 +77,87 @@ describe('components/notification/Notification', () => {
         },
     ].forEach(({ overflowOption, expectedClass }) => {
         test(`should render a notification with ${expectedClass} styling when passed the ${overflowOption} overflow option`, () => {
-            const component = mount(<Notification overflow={overflowOption}>test</Notification>);
+            const component = getWrapper({ overflow: overflowOption });
 
             expect(component.find('div.notification').hasClass(expectedClass)).toBe(true);
         });
     });
 
-    test('should call the onClose when the close button is clicked', () => {
-        const closeMock = sinon.spy();
+    test('should not set timeout if clickOnClose is set', () => {
+        getWrapper({ clickOnClose: true });
+        expect(setTimeout).toHaveBeenCalledTimes(0);
+    });
 
-        const component = mount(
-            <div>
-                <Notification onClose={closeMock}>test</Notification>
-            </div>,
-        );
+    test('should call the onClose when the close button is clicked', () => {
+        const closeMock = jest.fn();
+        const component = getWrapper({ onClose: closeMock });
 
         const closeBtn = component.find('button');
         closeBtn.simulate('click');
 
-        expect(closeMock.called).toBe(true);
+        expect(closeMock).toHaveBeenCalledTimes(1);
     });
 
-    test('should only call onClose once when the close button is clicked and the short duration expires', () => {
-        const closeMock = sinon.spy();
-
-        const component = mount(
-            <Notification duration="short" onClose={closeMock}>
-                test
-            </Notification>,
-        );
+    test('should call onClose when Escape key is pressed', () => {
+        const mockEvent = {
+            key: 'Escape',
+            preventDefault: jest.fn(),
+            stopPropagation: jest.fn(),
+        };
+        const closeMock = jest.fn();
+        const component = getWrapper({ onClose: closeMock });
+
+        component.simulate('keyDown', mockEvent);
```

Comment:
I've been told that we don't usually `simulate` events. Not sure if that wants to be applied here as well.